src/libs/PersonalDetailsUtils.ts

```diff
@@ -158,6 +158,7 @@ function getPersonalDetailsOnyxDataForOptimisticUsers(newLogins: string[], newAc
             login,
             accountID,
             displayName: LocalePhoneNumber.formatPhoneNumber(login),
+            isOptimisticPersonalDetail: true,
```

Comment:
Why are we adding a new property instead of reusing the `isPending` we use everywhere?

Reply:
Oh damn, did not notice that. Not sure why we picked a different name than the rest. Anyway 👍